php/libraries/Utility.class.inc

```diff
@@ -429,24 +447,33 @@ class Utility
      *
      * @return array If $var is an array, var, otherwise an array containing $var
      */
-    static function toArray($var): array
+    static function toArray($var)
     {
+        // TODO Uncomment the exception code below after LORIS 21 is released.
+        //throw new DeprecatedException(
+        //    'Utility::toArray() is deprecated and should not be used. ' .
+        //    'Instead use Utility::associativeToNumericArray() for converting ' .
+        //    'arrays. Scalar values cannot be passed to this function.'
+        //);
+        error_log(
+            'Utility::toArray() will be deprecated in LORIS ' .
+            'version 21. Please use Utility::associativeToNumericArray() ' .
```

Comment:
I merged this anyways, but in the future, I wouldn't specify the release number but just say "in a future release" to have a little more flexibility/just in case something comes up and it doesn't get done right away.